packages/babel-plugin-i18n-calypso/src/index.js

```diff
@@ -56,6 +56,31 @@ const DEFAULT_HEADERS = {
  */
 const DEFAULT_DIR = 'build/';
 
+/**
+ * List of translate function to be handled for string extraction.
+ *
+ * @type {string[]}
+ */
+const TRANSLATE_FUNCTION_NAMES = [ '__', '_n', '_nx', '_x', 'translate' ];
```

Comment:
Could we combine these 3 constants into one?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/39280/commits/ef9c5a27ade9f2c004fc8b2573e14cf10865ff70